Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -533,7 +533,21 @@ ThrowCompletionOr<PlainMonthDay*> calendar_month_day_from_fields(GlobalObject& g
     return static_cast<PlainMonthDay*>(month_day_object);
 }
 
-// 12.2.26 FormatCalendarAnnotation ( id, showCalendar ), https://tc39.es/proposal-temporal/#sec-temporal-formatcalendarannotation
+// 12.2.26 MaybeFormatCalendarAnnotation ( calendarObject, showCalendar ), https://tc39.es/proposal-temporal/#sec-temporal-maybeformatcalendarannotation
+ThrowCompletionOr<String> maybe_format_calendar_annotation(GlobalObject& global_object, Value calendar_object, StringView show_calendar)
```

Comment:
`calendar_object` should be an `Object&` (`Object const&`?)

Reply:
Actually, I was wrong about this - the spec says it must be an object, but it's called with undefined in at least one place. Will file a spec fix.